contents/handbook/company/communication.md

```diff
@@ -93,7 +93,7 @@ Use video calls if you find yourself going back and forth in an issue/via email
 1. Try to have your video on at all times because it's much more engaging for participants. Having pets, children, significant others, friends, and family visible during video chats is encouraged - please introduce them!
 1. As a remote company we are always striving to have the highest fidelity, collaborative conversations. Use of a headset with a microphone, is strongly recommended - use your company card if you need.
 1. Always advise participants to mute their mics if there is unnecessary background noise to ensure the speaker is able to be heard by all attendees.
-1. You should take notes of the points and to-dos during the meeting. Being able to structure conclusions and follow up actions in real time makes a video call more effective than an in-person meeting. If it is important enough to schedule a meeting, it is important enough to have take notes.
+1. You should take notes of the points and to-dos during the meeting. Being able to structure conclusions and follow-up actions in real time makes a video call more effective than an in-person meeting. If it is important enough to schedule a meeting, it is important enough to have taken notes.
```

Comment:
- The term "follow up" should be used as a verb. In this context, it would be more appropriate to use the adjective "follow-up". See: [Follow up, Follow-up, Followup](https://www.grammarly.com/blog/follow-up-followup/).
- The past participle "taken" should be used in this context.